torch/csrc/cuda/Module.cpp

```diff
@@ -921,6 +921,9 @@ static void registerCudaDeviceProperties(PyObject* module) {
       .def_readonly(
           "max_threads_per_multi_processor",
           &cudaDeviceProp::maxThreadsPerMultiProcessor)
+#if USE_ROCM
+      .def_readonly("warp_size", &hipDeviceProp_t::warpSize)
+#endif // USE_ROCM
```

Comment:
warpSize attribute exists for both cuda and rocm.  Let's avoid USE_ROCM where we can.

```suggestion
      .def_readonly("warp_size", &hipDeviceProp_t::warpSize)
```